pandas/tests/series/test_analytics.py

```diff
@@ -1625,7 +1625,7 @@ def test_search_sorted_datetime64_list(self):
     def test_searchsorted_sorter(self):
         # GH8490
         s = Series([3, 1, 2])
-        r = s.searchsorted([0, 3], sorter=np.argsort(s))
+        r = s.searchsorted([0, 3], 'left', np.argsort(s))
```

Comment:
Doesn't that come into conflict with your suggestion about adding `**kwargs` to `searchsorted`?